src/docs/grains/grain_persistence/relational_storage.md

```diff
@@ -93,122 +93,121 @@ public class AdoNetGrainStorageOptions
+This is teh case so that you can choose a serialization format, e.g. [Simple Binary Encoding (SBE)](https://github.com/real-logic/simple-binary-encoding) and implements
 [IStorageDeserializer](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/IStorageDeserializer.cs) and [IStorageSerializer](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/IStorageSerializer.cs).
-The built-in serializers have been built using this method. The [OrleansStorageDefault(De)Serializer](https://github.com/dotnet/orleans/tree/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider) can be used as examples on how to implement other formats.
+The built-in serializers have been built using this method. The [OrleansStorageDefault(De)Serializer](https://github.com/dotnet/orleans/tree/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider) can be used as examples of how to implement other formats.
 
 When the serializers have been implemented, they need to be added to the `StorageSerializationPicker` property in [AdoNetGrainStorage](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/AdoNetGrainStorage.cs).
-This is an implementation of [IStorageSerializationPicker](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/IStorageSerializationPicker.cs). By default
-[StorageSerializationPicker](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/StorageSerializationPicker.cs) will be used. And example of changing data storage format
+Here is an implementation of [IStorageSerializationPicker](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/IStorageSerializationPicker.cs). By default,
+[StorageSerializationPicker](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/StorageSerializationPicker.cs) will be used. An example of changing the data storage format
 or using serializers can be seen at [RelationalStorageTests](https://github.com/dotnet/orleans/blob/master/test/Extensions/TesterAdoNet/StorageTests/Relational/RelationalStorageTests.cs).
 
-Currently there is no method to expose this to Orleans application consumption as there is no method to access the framework created [AdoNetGrainStorage](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/AdoNetGrainStorage.cs).
+Currently, there is no method to expose this to the Orleans application as there is no method to access the framework-created [AdoNetGrainStorage](https://github.com/dotnet/orleans/blob/master/src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/AdoNetGrainStorage.cs).
```

Comment:
@veikkoeeva, @voipengineer asks:
> 131 - what is "this" referring to?

How do you think we should rephrase this? Perhaps "Current, there is no method to expose the serialization picker to the application as there is no method to access the created [AdoNetGrainStorage]"?

Reply:
Looking to these later today (takes at least 12 hours :)).

Reply:
What @ReubenBond wrote looks reasonable to me.

Context: This comes from before 1.5 era if I remember correctly. There was no DI back then. I do not know if this would be easy to do today. Even if it were, I would like to align with https://github.com/dotnet/orleans/pull/6936.